### PR TITLE
Style-fixes

### PR DESCRIPTION
**Pull request by THEjacob1000**
### Description
This pull request addresses several key issues that were affecting user experience and accessibility. The following are the major changes included:

### Fixes and Improvements:
1. **Styling Adjustments:**
   - Aligned elements according to the project manager's specifications to enhance visual consistency across pages.
   - Improved accessibility features to comply with WCAG guidelines, enhancing usability for users with disabilities.

2. **Searchbar Bug Fix:**
   - Resolved an issue where the searchbar was not allowing navigation due to a JavaScript error. Users can now use the searchbar to navigate the site effectively.

### CODE REVIEW

File: src/app/coins/[coinId]/page.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ const Page = ({ params }: { params: { coinId: string } }) => {
   const copyToClipboard = async (text: string) => {
     try {
       await navigator.clipboard.writeText(text);
-      console.log("Link copied to clipboard");
+      // console.log("Link copied to clipboard");
```
Comment (Basiol97) [inline src/app/coins/[coinId]/page.tsx]: Delete please

File: src/app/coins/[coinId]/page.tsx [hunk 2]
```diff
@@ -63,7 +63,7 @@ const Page = ({ params }: { params: { coinId: string } }) => {
           `/api/coinData?coinId=${coinId}`
         );
         const data = response.data;
-        console.log(data);
+        // console.log(data);
```
Comment (Basiol97) [inline src/app/coins/[coinId]/page.tsx]: Delete please

**Review by Basiol97 — APPROVED** [review]
Minor changes were made, primarily removing commented console.log statements.

File: src/app/coins/[coinId]/page.tsx (hunk 1 above)
Comment (THEjacob1000) [reply src/app/coins/[coinId]/page.tsx]: Done

File: src/app/coins/[coinId]/page.tsx (hunk 2 above)
Comment (THEjacob1000) [reply src/app/coins/[coinId]/page.tsx]: Done

**Pull request merged**